packages/swarm-components/src/TogglePill.jsx

```diff
@@ -1,24 +1,38 @@
+// @flow
 import * as React from 'react';
 
 export type Props = {
+  /**
+   * Indicates whether the toggle is selected
+   */
   checked: boolean,
+  /**
+   * Indicates whether the toggle is disabled
+   */
   disabled?: boolean,
+  /**
+   * Text label of the content
+   */
   label?: string,
-  name: string
+  /**
+   * Text label of the content, applied if label is not set
+   */
+  children?: any
```

Comment:
Consider using ` children?: React.ReactElement<*>` as the type? This enforces a single child node and potential type errors

Reply:
`children` should generally be `React.Node` (or `React$Node`): https://flow.org/en/docs/react/children/

I believe `React.ReactElement` won't allow primitives like text or numbers